<?xml version="1.0"?>
<launch>
    <node name ="xian_dj_retractable_platform_params_server" pkg="xian_dj_retractable_platform_param_pkg" type="xian_dj_retractable_platform_params_server" />
    <node name ="xian_dj_retractable_platform_base_electric" pkg="xian_dj_retractable_platform_control_pkg" type="xian_dj_retractable_platform_base_electric.py" />
    <node name ="xian_dj_retractable_platform_control" pkg="xian_dj_retractable_platform_control_pkg" type="xian_dj_retractable_platform_control" />
    <node name ="xian_dj_retractable_platform_back_end" pkg="xian_dj_retractable_platform_control_pkg" type="xian_dj_retractable_platform_back_end" />
    <!-- <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
    </include> -->

</launch>